apps/shared/account/account.service.ts

```diff
@@ -25,15 +33,21 @@ export class AccountService {
   /** The display name for the user is available. */
   displayName: string | undefined;
   /** The current accounts github token, if available. */
```

Comment:
Comment needs an update. Also shouldn't it be `null` if we are not authenticated?

Reply:
I am not sure what you mean here.

Reply:
Unless I miss something, the property no longer is the token- but a GitHub octokit API instance. The comment states that the property is the "token".

Also right now you always initialize `Octokit`. Would it make sense to only do it when authenticated?